esx_service/cli/vmdkops_post_update.11.1.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -------------------------------------------------------------------------------------------
+# Updates ESX with vSphere Docker Volume Service 0.11 (and earlier)
+#  to 0.11.1 and further
+#
+# This code expects to be places in  /usr/lib/vmware/vmdkops/bin/ to pick up correct modules.
```

Comment:
Sure, we can, but I think I just need to drop the comment since the file is now in the bin in the proper place . I'll drop the comment